metapool/sample_sheet.py

```diff
@@ -1575,6 +1709,8 @@ def demux_sample_sheet(sheet):
             sheet.Contact['Sample_Project'].isin(projects)].reset_index(
             drop=True)
 
+        # TODO: How should the SAMPLE_CONTEXT_KEY section be handled for reps?
```

Comment:
Ideally we'd copy the SAMPLE_CONTEXT_KEY into each replicate, removing the entries aren't present in that replicate's [Data] section. It might be more expedient however to copy the entire section into each replicate sample-sheet.

The demuxed sheets are really just for seqpro's benefit; it's our command-line tool that generates prep-info files from sample-sheets and other metadata. Ideally we'd pass the metadata directly to the functions seqpro calls and skip the need for creating replicate sample-sheets entirely.

Reply:
Done ... ugh, that hurt!